### DISCUSSION TITLE
Is it possible to host a project of this template on workers.dev

### DISCUSSION THREAD

**Discussion by hypzz** [kriasoft/react-starter-kit#2046] [Q&A] [answered]
It seems cloudflare zone id is required for deployment. Does that mean that a custom domain is required?

I don't want to attach a custom domain to my project just yet. Is it possible to use workers.dev for testing/prototyping stage?

**Comment by koistya** [comment, marked as answer]
@hypzz yes, just remove `zone_id` from `wrangler.toml` and add `workers_dev = true`:

https://github.com/kriasoft/react-starter-kit/blob/a131f504916cefa46fcd1dd4ff66e5d753970f24/edge/wrangler.toml#L13C1-L15

#### Reference

https://developers.cloudflare.com/workers/wrangler/configuration/

**Reply by hypzz** [reply]
Thank you!
